src/libraries/System.Console/tests/ReadAndWrite.cs

```diff
@@ -194,6 +194,25 @@ public static async Task OutWriteAndWriteLineOverloads()
         }
     }
 
+    [Fact]
+    [PlatformSpecific(TestPlatforms.iOS | TestPlatforms.MacCatalyst | TestPlatforms.tvOS)]
+    public void TestConsoleWrite()
```

Comment:
Would it be better to align this test with the code snippet and example output and incorporate `\n`
```
Console.Write ("A");
Console.Write ("B");
Console.WriteLine ("C");

Console.Write ("D\nd");
Console.Write ("E\ne");
Console.WriteLine ("F");
```